examples/stm32/f0/stm32f072-discovery/dac-simple/README

```diff
@@ -0,0 +1,6 @@
+------------------------------------------------------------------------------
+README
+------------------------------------------------------------------------------
+
+Simple DAC Test example
```

Comment:
Please expand on what this actually does.  Even better is to combine adc-dac-print sort of thing, so you can with a simple wire externally see both peripherals actually working.  needing a scope to see this working is problematic for many people.